src/config/config.php

```diff
@@ -90,6 +90,11 @@
         'composer' => \Folklore\GraphQL\View\GraphiQLComposer::class,
     ],
 
+    /*
+     * Whether or not to fire the 'TypeAdded' event. You can set this to false if you don't use it in your application.
```

Comment:
All other comments in this file try to keep line length to a limit. I thing it's more readable to break the line between the sentences.

Reply:
Fixed